manifests/user_setup.pp

```diff
@@ -0,0 +1,53 @@
+# This class hould be considered private
+#
```

Comment:
It would be useful to have a blurb discussing why `user/group` management needs to be handled before the package is installed.

Reply:
We had issues with using a custom home directory

Reply:
I meant a blurb as a comment on the class.